services/web/client/source/class/qxapp/component/widget/NodeView.js

```diff
@@ -273,6 +274,13 @@ qx.Class.define("qxapp.component.widget.NodeView", {
 
     __attachEventHandlers: function() {
       this.__blocker.addListener("tap", this.__inputNodesLayout.toggleCollapsed.bind(this.__inputNodesLayout));
+
+      qx.event.message.Bus.getInstance().subscribe("maximizeIframe", msg => {
```

Comment:
what happens if there are multiple nodes with iframes ? they will all trigger when any of the iframes maximizes

Reply:
this problem would not have a visible effect ... I was thinking of the node-input panels ... they exist inside each node, right? and if they all listen to this global message they will act even when the node is not in view ... which seems not an ideal design ...

Reply:
Added some code to stop listening to "maximizeIframe" message when the widget is not visible.